int32 publish_count
bool nook_status
